.ci_support/osx_.yaml

```diff
@@ -0,0 +1,18 @@
+MACOSX_DEPLOYMENT_TARGET:
+- '10.9'
+c_compiler:
+- clang
+c_compiler_version:
+- '9'
```

Comment:
is this conda-forge's clang? then can we use 10?

Reply:
Not only in this recipe but you would need to update the pinnings at https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/ab8ca327dd58354b750e9d01a7daea287975c82e/recipe/conda_build_config.yaml#L12

Reply:
(and yes, this is conda-forge's clang)